### DISCUSSION TITLE
Document .close() method on py-splashscreen

### DISCUSSION THREAD

**Discussion by MarcSkovMadsen** [pyscript/pyscript#1501] [Proposals]
I was trying to use [py-splashscreen](https://docs.pyscript.net/latest/reference/plugins/py-splashscreen.html). I would like it not to autoclose.

That was explained nicely on the page and worked great. But the natural next question is how do hide it. This is not documented.

Please document.

I found out by using the browser console to inspect the element. There is a `.close` method I can use. Also from Python.

**Comment by FabioRosado** [comment]
When you say that you would like to hide it. Do you mean to disable it completely?

If that's the case we have it [documented in the reference guide](https://docs.pyscript.net/latest/reference/plugins/py-splashscreen.html#disabling-the-splashscreen)

**Comment by MarcSkovMadsen** [comment]
Nope. I would like to close it programmatically, from Python when all the initial code has executed.

The .close method is not documented.
